legal-api/src/legal_api/resources/v2/business/business_digital_credentials.py

```diff
@@ -137,68 +158,146 @@ def send_credential(identifier, credential_type):
     issued_credential = DCIssuedCredential(
         dc_definition_id=definition.id,
         dc_connection_id=connection.id,
-        credential_exchange_id=response['credential_exchange_id']
+        credential_exchange_id=response['cred_ex_id'],
+        # TODO: Add a real ID
+        credential_id='123456'
```

Comment:
can we get rid of the hardcoded id now?

Reply:
I have an update that will properly generate the ID.